test/e2e/install_test.go

```diff
@@ -45,7 +47,23 @@ func checkIfInstalled() bool {
 }
 
 func TestInstall(t *testing.T) {
-	if !checkIfInstalled() {
-		commands.Install(nil, nil)
+	namespace := "default"
+	if !checkIfInstalled(namespace) {
+		args := commands.InstallFlags{
+			ControllerName: common.DefaultControllerDeploymentName,
+			UIName:         common.DefaultUiDeploymentName,
+			Namespace:      namespace,
+			ConfigMap:      common.DefaultConfigMapName(common.DefaultControllerDeploymentName),
+			//TODO(shri): Use better defaults that don't need Makefiles
```

Comment:
Yes, we should fall back on some sane defaults in the context that things are not built via make. For example, kubernetes go-client falls back on some default values in their version package:

https://github.com/kubernetes/client-go/blob/master/pkg/version/version.go
https://github.com/kubernetes/client-go/blob/master/pkg/version/base.go